xcode-install.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   # CLI parsing
-  spec.add_dependency 'claide', '>= 0.9.1', '< 1.1.0'
```

Comment:
I made this change I still get the `Unable to activate xcode-install-2.8.0, because claide-1.1.0 conflicts with claide (>= 0.9.1, < 1.1.0) (Gem::ConflictError)` message

Reply:
@triplef any clue? ☝️ 

I think the underlying problem is the installation into the system's gems instead of local bundled gems 🤕

Reply:
Fair enough. Merging this in!